source/elements/oneDNN/source/graph/ops/HardSigmoid.rst

```diff
@@ -0,0 +1,71 @@
+.. SPDX-FileCopyrightText: 2020-2023 Intel Corporation
+..
+.. SPDX-License-Identifier: CC-BY-4.0
+
+.. include:: ../../replacements.inc.rst
+
+
+HardSigmoid
+###########
+
+HardSigmoid operation applies the following formula on every element of \src
+tensor (the variable names follow the standard @ref dev_guide_conventions):
+
+.. math::
+
+    \dst = \max(0, \min(1, \alpha \src + \beta))
+
+Operation Attributes
+********************
+
++--------------+-------------+------------+--------------+-------------+
+|  Attribute   | Description | Value Type |  Supported   | Required or |
+|     Name     |             |            |    Values    |  Optional   |
++==============+=============+============+==============+=============+
+| |attr_alpha| | Alpha in    | f32        | Arbitrary    | Required    |
+|              | the         |            | f32 value    |             |
+|              | formula.    |            |              |             |
++--------------+-------------+------------+--------------+-------------+
+| |attr_beta|  | Beta in     | f32        | Arbitrary    | Required    |
```

Comment:
Please use ``` :math:`\alpha` ``` and ``` :math:`\beta` ```, so all symbols are rendered the same.
The same applies to backward document.

Reply:
Thanks for reminder. Fixed now.